2021/OffSeason2021/src/main/cpp/subsystems/Intake.cpp

```diff
@@ -43,8 +43,8 @@ Intake::Intake()
 
     // Initialize Variables
     frc2135::RobotConfig *config = frc2135::RobotConfig::GetInstance();
-    config->GetValueAsDouble("IN_AcquireSpeed", m_acquireSpeed, 0.8);
-    config->GetValueAsDouble("IN_ExpelSpeed", m_expelSpeed, -0.8);
+    config->GetValueAsDouble("IN_AcquireSpeed", m_acquireSpeed, 1.0);

```

Comment:
This looks good! Quick question though. Weren't there supposed to be two intake acquire modes because intake action and shooting action require two different intake speeds?